### PR TITLE
Re-activate coala-ci

### PR DESCRIPTION
**Pull request by jayvdb**
Closes https://github.com/coala/projects/issues/13

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +10,9 @@ before_script:
 
 # Assume bundler is being used, therefore
 # the `install` step will run `bundle install` by default.
-script: ./ci/cibuild
+script:
+  - ./ci/cibuild
+  - docker run -v=$(pwd):/app --workdir=/app coala/base coala --ci
```
Comment (sims1253) [inline .travis.yml]: is the /app actually working in this case? Shouldn't it point to the projects directory?

File: .coafile [hunk 2]
```diff
@@ -1,7 +1,8 @@
 [global]
-bears = coalaBear
+bears = coalaBear, InvalidLinkBear
 files = **.yml, **.html, **.md, **.js, **.css, **.json
 ignore = vendor/**, resources/vendors/**, _site/**
+link_ignore_regex = (twitter.com|githubusercontent.com|github.com|gitlab.com|withgoogle.com|clbin.com)
```
Comment (sims1253) [inline .coafile]: why do we want to ignore all of these? github links for example might point to our wiki so they should work.

File: reports/satwikkansal.md [hunk 3]
```diff
@@ -292,6 +292,6 @@ Apart from that, there's also scope to extract useful information from meta file
 
 #### Thanks giving!
 
-Thanks a lot to all my fellow coalians for making my so far wonderful journey with coala. It's been around 9 months since I started contributing to coala and I've learned a lot till now. A very very special thanks to [Adhityaa](https://gihub.com/adtac) for helping me constantly throughout the summer despite of his own internship. He was quick in doing critical code reviews and absolutely a fun mentor to work with (not to mention the interesting conversations between [Adrian](https://github.com/Adrianzatreanu) and him during the initial weekly meetings).
+Thanks a lot to all my fellow coalians for making my so far wonderful journey with coala. It's been around 9 months since I started contributing to coala and I've learned a lot till now. A very very special thanks to [Adhityaa](https://github.com/adtac) for helping me constantly throughout the summer despite of his own internship. He was quick in doing critical code reviews and absolutely a fun mentor to work with (not to mention the interesting conversations between [Adrian](https://github.com/Adrianzatreanu) and him during the initial weekly meetings).
```
Comment (sims1253) [inline reports/satwikkansal.md]: Is the only change here - -> + ?
If so, why? we now use *, - and + for list points in this file.

File: .coafile (hunk 2 above)
Comment (jayvdb) [reply .coafile]: because they return 404 for any bots.

File: .travis.yml (hunk 1 above)
Comment (jayvdb) [reply .travis.yml]: See User Docker documentation

File: reports/satwikkansal.md (hunk 3 above)
Comment (jayvdb) [reply reports/satwikkansal.md]: The change here is gihub.com->github.com

Comment (neelanshsahai) [inline reports/satwikkansal.md]: The first sentence seems to be Grammatically incorrect.

Comment (jayvdb) [reply reports/satwikkansal.md]: That is not part of the issue this PR is solving, so it does not belong in this PR.

File: .coafile (hunk 2 above)
Comment (Makman2) [reply .coafile]: then please add a comment mentioning that :+1:

**Comment by Makman2** [comment]
ack 2d4e987

File: .coafile (hunk 2 above)
Comment (jayvdb) [reply .coafile]: Here is the removal https://github.com/coala/projects/commit/6e75373176735

**Comment by Makman2** [comment]
ack 6b8b4c8

**Comment by Makman2** [comment]
ack 20dcfaf

**Comment by Makman2** [comment]
@rultor merge

**Pull request merged**